modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -21,6 +21,8 @@
 @{
     @defgroup gapi_filters Graph API: Image filters
     @defgroup gapi_colorconvert Graph API: Converting image from one color space to another
+    @defgroup gapi_feature Graph API: Image Feature Detection
+    @defgroup gapi_shape Graph API: Image Structural Analysis and Shape Descriptors
```

Comment:
gapi_shape, but Structural Analysis and Shape Descriptors. How it was called in one word in the textbook? I don't remember.

Reply:
Ok. It is the same as ocv defgroup.

Reply:
Yeah, correct